### DISCUSSION TITLE
Populations jumping when dynamic landscape added

### DISCUSSION THREAD

**Discussion by bs17smh** [RangeShifter/RangeshiftR-tutorials#34] [Q&A] [answered]
Hello, 

When using the GUI version to test some simulations I notive on the year a dynamic landscape is introduced my population will disappear from where they were and appear at a random part of the map. I'm unsure if this is just a graphical bug as they fill back into where they were very quickly. However as they remain in the place they jumped to (a distance they shouldn't be able to reach), i'm worried its going to throw off my results. I've attached maps, one before the landscape is added and another at the year it is added

any help appreciated, cheers
![Sim0_land0_rep0_yr25_rs0](https://user-images.githubusercontent.com/64581667/182565909-55fe85cb-1661-4d89-a11b-f0f4f9740cd4.jpeg)
![Sim0_land0_rep0_yr50_rs0](https://user-images.githubusercontent.com/64581667/182566156-36d6f6df-acc7-44f0-9266-1a0d9a14cbc9.jpeg)

**Comment by JetteReeg** [comment, marked as answer]
Hi, 
without any further information about your settings, it is difficult to answer your question. Have you read the manual regarding dynamic landscapes? In the manual it says 

> The identity of patches is not cross-checked between changes, and care must therefore be taken to ensure consistency; otherwise, a patch (and its resident population) can jump to a distant location or be split into two or more disjunct parts, with unpredictable and possibly weird consequences. (p 13)

If you need further help, it would be great if you could provide us with more specifics about you settings or give a minimal working example which reproduce the potential error.

Best, Jette

**Reply by bs17smh** [reply]
I think that answers my question perfectly! Sorry, not read the manual in a while and should've gone back to it first

best,
Sam
